### PR TITLE
Addition Section "Network Providers" on the Staking Page

### LINKED ISSUE #4041: Add info about network provider to Staking page
Make the user aware of network providers and that decentralization also involves provider location and reward allocation by validators. Mention that some providers might not support PoS; consequently, it is important to double-check validators accordingly. 

- Mention the Polkawatch app as a tool to determine how your reward payout is decentralized in terms of geolocation and network provider.

### PR DESCRIPTION
**Pull request by filippoweb3**
See linked issue.

### CODE REVIEW

**Comment by filippoweb3** [comment]
@DrW3RK added some info, not sure if I am going too far and if I am allowed to mention everything in the paragraph. Let me know what you think.

File: docs/learn/learn-staking.md [hunk 1]
```diff
@@ -258,6 +258,14 @@ To maximize rewards and minimize risk, one could select those validators that:
   to their website and contact details),
 - and have not been slashed (meaning that their on-chain behavior is genuine).
 
+#### Network Providers
+
+Most validator nodes run on cloud service providers that guarantee high hardware specifications and all safety measures in the case, for example, there is a shortage in power supply. Keep in mind that if a validator is supposed to be online producing blocks and the internet connection drops because there is a cutoff in electricity, if the node is run in-house and does not have a backup, that validator will be slashed. As a consequence, the stake of that validator, as well as the stake of all nominators, will be partially or even entirely confiscated (depending on the gravity of the situation). Cloud services reduce such risks and provide a way of decentralizing the locations of the nodes. Validators can live in one part of the world and set up a node in a different region.
```
Comment (profradha) [inline docs/learn/learn-staking.md]:
```suggestion

For successful operation, a Validator node should always be ensured to meet the required [software, hardware, and network bandwidth specifications](../maintain/maintain-guides-how-to-validate-polkadot#reference-hardware). 
Understandably, most of the validator nodes run on cloud service providers that guarantee high hardware specifications and high levels of availability and connectivity. Keep in mind that a validator in the active set is supposed to be fully online and available for producing blocks. If the active validator node goes offline due to network interruptions or a power outage, that validator might be subject to [slashing due to unresponsiveness](learn-staking-advanced#unresponsiveness). As [Polkadot's block production mechanism] is reasonably resilient to a small proportion of validators going offline, no slashing is imposed until 10% of the validators in the active set go offline. Hence, if multiple nodes are running on a single cloud service provider and go offline simultaneously due to an outage or due to a change in their terms and conditions policy regarding the support of Proof-of-Stake (PoS) operations, the offline validators and all the nominators backing them can be slashed up 7% of their stake on Polkadot.
```

File: docs/learn/learn-staking.md [hunk 2]
```diff
@@ -258,6 +258,14 @@ To maximize rewards and minimize risk, one could select those validators that:
...
+
+However, cloud services might not support Proof-of-Stake (PoS) operations in their Terms and Conditions Policy, depending on the country they are based in. In the case of conflict between the provider and PoS nodes, the provider can cut off such nodes. This can significantly affect the network, especially if many nodes are running on that provider. For example, on the 2nd of November 2022, a thousand validators on Solana (40% of the total number of nodes) went offline because Hetzner Online GmbH stated they do not support cryptocurrency activities.
```
Comment (profradha) [inline docs/learn/learn-staking.md]: Removing this section
```suggestion
```

File: docs/learn/learn-staking.md [hunk 3]
```diff
@@ -258,6 +258,14 @@ To maximize rewards and minimize risk, one could select those validators that:
...
+
+You must check if your nominations use cloud service providers that allow for Proof-of-Stake operations. You can contact them via email providers, and they should get back to you with more information about this topic. Additionally, you can connect your stash account to the [Polkawatch app](https://polkawatch.app/). The app will show your rewards earned in the past 60 eras divided by network provider and country. You will be able to see networks used by each validator and verify if your validators are using providers who support PoS. This is also a great tool to explore how decentralized your nominations are and act accordingly.
```
Comment (profradha) [inline docs/learn/learn-staking.md]:
```suggestion
Hence, it is recommended that you check if you are nominating the validator nodes that are running on cloud service providers, and if they do, check if they allow for Proof-of-Stake operations. 

:::tip

You can connect your stash account to the [Polkawatch app](https://polkawatch.app/). The app will show your rewards earned in the past 60 eras divided by network provider and country. You will be able to see networks used by each validator and verify if your validators are using providers who support PoS. This is also a great tool to explore how decentralized your nominations are and act accordingly.

:::

```

**Review by DrW3RK — CHANGES_REQUESTED** [review]
Thanks for taking the initiative for adding this to the Staking doc. A very important piece of info in the benefit of nominators. Polishing the writeup a bit by providing links to the relevant docs in the Wiki. Also, slashing due to unresponsiveness is ceiled at 7% and it does not result in the entire stake being confiscated. Probably better to back the Solana/Hetzner incident with an article from a reliable source or leave out mentioning them altogether.

**Comment by filippoweb3** [comment]
@DrW3RK thanks for checking this paragraph! I added some minor edits.

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**